build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
In all files, except for `.git*`, please add a copyright header. 

You can grab one from [here](https://github.com/spine-examples/hello/blob/master/build.gradle), or any other file in our Spine examples.